### DISCUSSION TITLE
Using Deta micros with a MEN/MERN Stack application

### DISCUSSION THREAD

**Discussion by tyebean** [deta/cloud-docs#719] [Help]
Hey! I'm currently trying to integrate deta micros with a project currently pinned on my github called Proclaim. Is there a resource of a simple crud app anywhere that will show the file structure necessary to pull this off? I'm so close to getting it to work but I'm stuck somewhere.

**Comment by sudo-subg** [comment]
This might help you https://docs.deta.sh/docs/base/node_tutorial.
